sdk/v1/client_test.go

```diff
@@ -679,3 +699,127 @@ func TestClient_GetTask(t *testing.T) {
 		})
 	})
 }
+
+func TestClient_GetTasks(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	pathToCheck := "v1/jobs/883c81fd-726d-4ea3-9db8-7e7c781a01cc/tasks"
+
+	reqHeaders := client.Headers{
+		IfMatch:          "*",
+		ServiceAuthToken: "",
+	}
+
+	Convey("Given clienter.Do doesn't return an error", t, func() {
+		body, err := json.Marshal(expectedTasks)
+		if err != nil {
+			t.Errorf("failed to setup test data, error: %v", err)
+		}
+
+		httpClient := newMockHTTPClient(
+			&http.Response{
+				StatusCode: http.StatusCreated,
+				Body:       io.NopCloser(bytes.NewReader(body)),
+				Header: http.Header{
+					"Etag": []string{testETag},
+				},
+			},
+			nil)
+
+		searchReindexClient := newSearchReindexClient(t, httpClient)
+
+		Convey("When search-reindexClient.GetTasks is called", func() {
+			respHeaders, tasks, err := searchReindexClient.GetTasks(ctx, reqHeaders, testJobID)
+			So(err, ShouldBeNil)
+
+			Convey("Then the expected tasks list is returned", func() {
+				So(tasks.TaskList, ShouldHaveLength, 2)
+				So(tasks.Count, ShouldEqual, 2)
+				So(tasks.Limit, ShouldEqual, 5)
+				So(tasks.Offset, ShouldEqual, 0)
+				So(tasks.TotalCount, ShouldEqual, 20)
+				So(tasks.TaskList[0].JobID, ShouldEqual, "883c81fd-726d-4ea3-9db8-7e7c781a01cc")
+				So(tasks.TaskList[0].TaskName, ShouldEqual, "zebedee")
+				So(tasks.TaskList[0].NumberOfDocuments, ShouldEqual, 10)
+				So(tasks.TaskList[1].JobID, ShouldEqual, "883c81fd-726d-4ea3-9db8-7e7c781a01cc")
+				So(tasks.TaskList[1].TaskName, ShouldEqual, "dataset-api")
+				So(tasks.TaskList[1].NumberOfDocuments, ShouldEqual, 20)
+			})
+
+			Convey("And an ETag is returned", func() {
+				So(respHeaders, ShouldNotBeNil)
```

Comment:
These two assertions (line 750 and 751) can be removed as they are duplicating the same checks on ETag entry

```suggestion
```